From 3cd2b7dc37de1ac6876e4cb0de8b73e8f81d844a Mon Sep 17 00:00:00 2001
From: Petr Salinger <Petr.Salinger@seznam.cz>
Date: Wed, 3 Sep 2008 09:28:20 +0200
Subject: [PATCH] fix FTBFS on GNU/kFreeBSD

The implementation of function pa_drop_caps() in caps.c does not match
prototype in caps.h.
---
 src/daemon/caps.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/src/daemon/caps.c b/src/daemon/caps.c
index 8b19838..fbe4471 100644
--- a/src/daemon/caps.c
+++ b/src/daemon/caps.c
@@ -131,9 +131,8 @@ int pa_limit_caps(void) {
     return 0;
 }
 
-int pa_drop_caps(void) {
+void pa_drop_caps(void) {
     pa_drop_root();
-    return 0;
 }
 
 #endif
